test/surface/test_poisson.cpp

```diff
@@ -72,22 +72,22 @@ TEST (PCL, Poisson)
 
 //  io::saveVTKFile ("bunny_poisson.vtk", mesh);
 
-  ASSERT_EQ (mesh.polygons.size (), 1051);
+  ASSERT_EQ (mesh.polygons.size (), 4828);
```

Comment:
Any reason why the magic numbers changed?

Please add a note saying that the test checks if the implementation changed, and not the implementation itself.

Reply:
Added the note.
These were the numbers being generated by the function.  I even tried an older version of the `bun0.pcd` file, it too yielded same results, not sure how these magic numbers were selected in the first place. Maybe like you suggested these numbers changed due to implementation changes?